### PR TITLE
Chart refactor

### PR DESCRIPTION
**Pull request by schalkventer**
screencapture-localhost-4000-2018-19-national-departments-defence-and-military-veterans-2018-11-04-21_02_30

image

### CODE REVIEW

File: _includes/components/department-budgets/ExpenditureChart/index.jsx [hunk 1]
```diff
@@ -109,11 +52,7 @@ export default function ExpenditureChart(props) {
             </div>
           </div>
           <div className="u-marginLeft30">
-            {dataset && buildDataset()}
-            {guide && buildGuide()}
-            {pdf && buildPdf()}
-            {excel && buildExcel()}
-            {cpi && buildCpi()}
+            <Linkslist listArray={linksListArray} />
```
Comment (longhotsummer) [inline _includes/components/department-budgets/ExpenditureChart/index.jsx]: For the other charts, we do this all server-side (in Jekyll). Can we not do the same here?

Comment (schalkventer) [reply _includes/components/department-budgets/ExpenditureChart/index.jsx]: @longhotsummer This is actually an oversight on my side. `ExpenditureChart` is actually redundant now and can be removed. Will do so now.

File: _layouts/homepage.html [hunk 2]
```diff
@@ -21,18 +21,13 @@ <h1 class="Page-title">The South African National Budget {{ page.financial_year
...
-          type="alert"
-          size="large"
-        %}
-        <span class="Alert-heading">
-          Please note
-        </span>
-      </div>
-      <div class="Alert-text">The full Medium Term Budget Policy Statement is <a href="http://www.treasury.gov.za/documents/mtbps/2018/default.aspx">available on National Treasury's website</a>. The MTBPS documents and data will be added to vulekamali.gov.za soon.</div>
+    <div class="u-marginBottom20">
+      {%
```
Comment (longhotsummer) [inline _layouts/homepage.html]: This goes against what we did in https://github.com/OpenUpSA/static-budget-portal/pull/405/files#diff-6d5885cbf9d64c48d9766289aff8666eR24 where you just included the raw HTML. I still think you're making life hard for yourself turning this into a component -- it's an unnecessary abstraction that doesn't buy you much, and essentially is a thin wrapper over really simple HTML. Your component now has a number of conditionals, which could be eliminated.

Comment (longhotsummer) [reply _layouts/homepage.html]: I think the second code set is easier to read -- I don't need to unwrap a layer of abstraction, and I can see that I can put plain old HTML wherever I like, including links (which was the issue last time). The more abstractions we put over "plain old HTML", the harder we have to think about extensibility, re-use, etc. It's okay to repeat HTML.

**Comment by longhotsummer** [comment]
LGTM.

**Pull request merged**